crates/bevy_ecs/src/query/iter.rs

```diff
@@ -0,0 +1,131 @@
+use crate::{
+    archetype::{ArchetypeId, Archetypes},
+    query::{Fetch, FilterFetch, QueryState, WorldQuery},
+    storage::{TableId, Tables},
+    world::World,
+};
+
+pub struct QueryIter<'w, 's, Q: WorldQuery, F: WorldQuery>
+where
+    F::Fetch: FilterFetch,
+{
+    tables: &'w Tables,
+    archetypes: &'w Archetypes,
+    query_state: &'s QueryState<Q, F>,
+    world: &'w World,
+    table_id_iter: std::slice::Iter<'s, TableId>,
+    archetype_id_iter: std::slice::Iter<'s, ArchetypeId>,
+    fetch: Q::Fetch,
+    filter: F::Fetch,
+    pub(crate) is_dense: bool,
```

Comment:
Is this modified outside of this module? If not maybe add a `pub(crate)` getter instead? Also you may want to use `pub(in crate::...)` if the scope from which it is accessed is more limited than the whole crate.

Reply:
pub(crate) actually wasn't needed here. probably an artifact of a previous impl. i just removed it.